docs/source/customize.rst

```diff
@@ -26,6 +26,10 @@ Input Options
   multiple files.
 ``spelling_word_list_filename=['spelling_wordlist.txt','another_list.txt']``
   Same as above, but with several files of correctly spelled words.
+``spelling_ignore_filename=['ignored_file']``
```

Comment:
The value of the new option is a list, so the name should be plural.

Reply:
Great point, this has been updated.